src/App.vue

```diff
@@ -7,50 +7,56 @@
 <script lang="ts">
 import Component from 'vue-class-component'
 import Vue from 'vue'
-import { Action, Getter } from 'vuex-class'
-import { RoomAction, Locale, GameName, RoomGetters } from '@/store/modules/room'
-import { AuthenticationActions, AuthenticationGetters, ExtendedJWT, JWT_KEY } from '@/store/modules/authentication'
+import { Action, Getter, namespace } from 'vuex-class'
+import { Namespaces } from '@/store'
 import { Api } from './store/modules/types'
 import { getWowsApiData } from '@/games/wows/api'
+import { GameName, Locale, RoomAction, RoomGetters } from '@/store/modules/room'
+import { AuthenticationActions, AuthenticationGetters, ExtendedJWT, JWT_KEY } from '@/store/modules/authentication'
+
+const authNamespace = namespace(Namespaces.AUTH)
 
-@Component({
-  name: 'TheApp'
-})
+  @Component({
+    name: 'TheApp'
+  })
 export default class TheApp extends Vue {
-  @Action(`room/${RoomAction.SET_GAME_NAME}`) setGameName!: (name: string) => void
-  @Action(`room/${RoomAction.SET_GAME_API}`) setGameApi!: (api: Api) => void
-  @Action(`room/${RoomAction.SET_LOCALE}`) setLocale!: (locale: Locale) => void
-  @Getter(`room/${RoomGetters.GAME_NAME}`) game!: GameName
-  @Action(`authentication/${AuthenticationActions.AUTHENTICATE}`) authenticate!: (token: string) => Promise<ExtendedJWT>
-  @Action(`authentication/${AuthenticationActions.CHECK_TOKEN_EXPIRY}`) checkTokenExpiry!: () => Promise<boolean>
-  @Action(`authentication/${AuthenticationActions.STORE_TOKEN}`) storeToken!: (token: string) => void
-  @Getter(`authentication/${AuthenticationGetters.JWT}`) jwt!: ExtendedJWT
+    @Action(`room/${RoomAction.SET_GAME_NAME}`) setGameName!: (name: string) => void
```

Comment:
Can we use namespace please instead of actions and getters? You have already done it for the auth can we do it for the room too? That way we have one approach at writing this.